examples/prescriptors/neat/train_prescriptor.py

```diff
@@ -151,13 +176,6 @@ def eval_genomes(genomes, config):
         # stringency zero. To achieve more interesting behavior, a different fitness
         # function may be required.
         new_cases = pred_df[PRED_CASES_COL].mean().mean()
```

Comment:
no longer need this line, since stringency has already been computed incrementally

Reply:
But stringency is a sum now, instead of a mean?